src/scripts/modules/footer/footer-template.html

```diff
@@ -20,13 +20,6 @@
     </div>
   </section>
 
-  <section class="connect">
```

Comment:
Jello! Should we remove all the code for `class="connect"` and `data-l10n-id="all-footer-connect"` as well? It looks like it's only used here.

Reply:
Yeah I think it's safe to remove it.  I did a search on the webview code:

```
karen@yachiyo:~/src/webview$ git grep '\<connect\>'
src/locale/en-US/dictionary.ftl:all-footer-connect = Connect
src/locale/pl/dictionary.ftl:all-footer-connect = Łącza
src/scripts/modules/footer/footer.coffee:      @$el.find('.connect').height(height)
src/scripts/modules/footer/footer.less:  > .connect, > .share {
src/scripts/modules/footer/footer.less:  > .connect {
```

Looks to me they can all be removed.